src/db_iter.cc

```diff
@@ -0,0 +1,161 @@
+#include "db_iter.h"
+
+#include "blob_storage.h"
+#include "db/arena_wrapped_db_iter.h"
+
+namespace rocksdb {
+namespace titandb {
+
+TitanDBIterator::TitanDBIterator(const TitanReadOptions &options,
```

Comment:
why move

Reply:
Don't even remember.. I'll just revert and see.